docs/03_design/api/openapi.yaml

```
@@ -408,19 +408,76 @@ components:
+              description: メッセージスキーマ SemVer (major 不一致で互換性警告)
+            presetId:
+              type: string
+              const: blur
+              description: VisualEffectPreset.id を指定
+            params:
+              $ref: '#/components/schemas/BlurParams'
+      discriminator:
+        propertyName: presetId
+        mapping:
+          rainbow: '#/components/schemas/RainbowParams'
+          flash:   '#/components/schemas/FlashParams'
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)